_data/v2_6/navbars/getting-started.yml

```diff
@@ -23,6 +23,8 @@ toc:
         path: /getting-started/kubernetes/installation/hosted/kubeadm/
       - title: Kubernetes Datastore Hosted Install
         path: /getting-started/kubernetes/installation/hosted/kubernetes-datastore/
+      - title: Canal/flannel Hosted Install
```

Comment:
We also need this stuff  in the master directory, no?

Reply:
Yes, I haven't copied over the new Canal page or manifests to the master yet.